config/samples/keda_v1alpha1_kedacontroller.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: keda.sh/v1alpha1
+kind: KedaController
+metadata:
+  name: keda
+  namespace: keda
+spec:
+  ###
+  # THERE SHOULD BE ONLY ONE INSTANCE OF THIS RESOURCE PER CLUSTER 
+  # with Name set to 'keda' created in namespace 'keda'
+  ###
+
+  ## Namespace that should be watched by KEDA Controller, 
+  # omit or set empty to watch all namespaces (default setting)
+  watchNamespace: ""
+
+  ## Logging level for KEDA Controller 
+  # allowed values: 'debug', 'info', 'error', or an integer value greater than 0, specified as string
+  # default value: info
+  logLevel: info
+
+  ## Logging time format for KEDA Controller
+  # allowed values: 'epoch', 'millis', 'nano', or 'iso8601'
+  # default value: epoch
+  logTimeFormat: epoch
```

Comment:
update the sample pls, `logTimeFormat` was removed, encoder was introduced instead: https://github.com/kedacore/keda/blob/main/BUILD.md#keda-operator-logging

There might be other occurencies of `logTimeFormat` in the rest of the PR, please double check.

Reply:
created a separate issue to track the update https://github.com/kedacore/keda-olm-operator/issues/45 (a separate PR is a preffered)